### DISCUSSION TITLE
Don't ignore .github directory on GitHub Pages

### DISCUSSION THREAD

**Discussion by cirosantilli** [community/community#52252] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The .github/* directory seems to be ignored from GitHub pages ,e.g.:  https://github.com/cirosantilli/test-gh-pages/tree/d289552f9cf67257115a3eab3d6dc43b61e2d897/.github leads to 404s to any files in the directory:

- https://cirosantilli.com/test-gh-pages/.github/index.html
- https://cirosantilli.com/test-gh-pages/.github/notindex.html
- https://cirosantilli.com/test-gh-pages/.github/FUNDING.yml

If I put something in `.github` on my gh pages branch, please show the thing I put there :pray:

Use case: automated directory listings on a static website.

**Comment by yoannchaudet** [comment]
We automatically exclude the `.git` and `.github` folders because typically they don't contain anything worse serving as a static website.

The exclusion logic lives in https://github.com/actions/upload-pages-artifact. You can open an issue there but I doubt we will remove these exclusions or make them customizable in the short term.

We already started leveraging `.github` for special routes which would have precedence over anything you would try to serve yourself.

**Reply by cirosantilli** [reply]
Thanks, created https://github.com/actions/upload-pages-artifact/issues/56 Which special routes BTW?

**Reply by yoannchaudet** [reply]
Non-documented ones related to Private Pages. A feature for Enterprise Cloud customers.
